nuxeo-core/nuxeo-core-test/src/test/resources/OSGI-INF/test-coldstorage-contrib.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.coldstorage.test">
+
+  <extension target="org.nuxeo.ecm.core.api.thumbnail.ThumbnailService"
+             point="thumbnailFactory">
+    <thumbnailFactory name="fakeThumbnailFactory"
+                      factoryClass="org.nuxeo.ecm.core.DummyThumbnailFactory" />
+  </extension>
+
+  <extension target="org.nuxeo.ecm.core.blob.BlobManager" point="configuration">
+    <blobprovider name="dummy">
```

Comment:
I thought you would keep the `coldstorage` one and remove the `dummy` one, relying `test-dummy-blob-provider.xml` to be deployed in the tests.

Reply:
OK, I see that you're using the "other" blobProvider for coldContent, so you could not define any `blobprovider` at all here I guess, and rely on both contributions:
- `test-blob-provider-inmemory.xml` -> ` <blobprovider name="other">`
- `test-dummy-blob-provider.xml` -> `<blobprovider name="dummy">`

Or I'm missing something.

Reply:
as we discussed i rollback/revert this part of using a dispatcher i did it to prepare some stuff on the nexts development when we will have the low implementation on S3 Glacier. But  i agree we don't need it right now